hibernate-core/src/test/java/org/hibernate/orm/test/querycache/QueryCacheTest.java

```diff
@@ -68,7 +76,8 @@
 		settings = {
 				@Setting(name = AvailableSettings.USE_QUERY_CACHE, value = "true"),
 				@Setting(name = AvailableSettings.CACHE_REGION_PREFIX, value = "foo"),
-				@Setting(name = AvailableSettings.USE_SECOND_LEVEL_CACHE, value = "true")
+				@Setting(name = AvailableSettings.USE_SECOND_LEVEL_CACHE, value = "true"),
+				@Setting(name = JdbcSettings.CONNECTION_PROVIDER, value = "org.hibernate.orm.test.querycache.QueryCacheTest$ProxyConnectionProvider")
 		}
 )
 public class QueryCacheTest {
```

Comment:
I tried running this test _without_ your fix, and it did not fail (again current `main` branch).

I think it's important that we have a test which does reliably fail and demonstrate the issue.

Reply:
I agree. The test was failing the day I opened the pull request. Maybe something changed. I will check that asap.

Reply:
Thanks